ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-master.yaml

```diff
@@ -61,3 +61,17 @@ tests:
   commands: TEST_FOCUS=Suite:openshift/conformance/parallel run-tests
   openshift_ansible:
     cluster_profile: gcp-ha
+- as: e2e-gcp-logging
+  commands: TEST_FOCUS=Suite:openshift/conformance/parallel run-tests
+  openshift_ansible:
+    cluster_profile: gcp-logging
+- as: e2e-gluster
+  commands: TEST_FOCUS=Suite:openshift/conformance/parallel/minimal run-tests
+  openshift_ansible_custom:
+    cluster_profile: aws-gluster
+- as: e2e-gcp-major-upgrade-master
```

Comment:
You're putting this into the `master` branch configuration so it seems like `-master` suffix is not necessary here

Reply:
Makes sense, done